src/components/KSlideout/KSlideout.vue

```diff
@@ -3,28 +3,55 @@
     <transition name="fade">
       <div
         v-if="isVisible"
-        class="panel-background"
-        @click="(event: any) => handleClose(event, true)"
+        :class="hasOverlay ? 'panel-background' : 'panel-background-transparent'"
       />
     </transition>
-    <transition
-      name="slide"
-    >
+    <transition name="slide">
       <div
         v-if="isVisible"
+        ref="slideOutRef"
         class="panel"
-        :class="{ isVisible: 'is-visible' }"
+        :class="{ 'is-visible': isVisible, 'border-styles': !hasOverlay }"
       >
+        <div class="k-slideout-header-content">
+          <div
+            v-if="hasBeforeTitle"
+            class="k-slideout-before-title"
+          >
+            <slot name="before-title" />
+          </div>
+
+          <!-- title -->
+          <div class="k-slideout-main-title">
+            <p
+              class="k-slideout-title"
+              data-testid="k-slideout-title"
+            >
```

Comment:
```suggestion
            <p
              class="k-slideout-title"
              data-testid="k-slideout-title"
              :title="title"
            >
```

issue: always offer the full text on hover of truncated strings